packages/material-ui/src/SwipeableDrawer/SwipeableDrawer.js

```diff
@@ -473,7 +476,7 @@ const SwipeableDrawer = React.forwardRef(function SwipeableDrawer(inProps, ref)
       // The ref may be null when a parent component updates while swiping.
       setPosition(
         getMaxTranslate(horizontalSwipe, paperRef.current) +
-          (disableDiscovery ? 20 : -swipeAreaWidth),
+          (disableDiscovery ? 1 : -1) * DRAG_STARTED_SIGNAL,
```

Comment:
The values are different, the first one is to hide the box-shadow.

```suggestion
          (disableDiscovery ? 15 : -DRAG_STARTED_SIGNAL,
```

Reply:
@oliviertassinari thanks for your fix. But it was 20 in previous version.

Reply:
True, I have changed it to 15 so it doesn't get confused with DRAG_STARTED_SIGNAL. It's a different value.